src/k8s-engine/lima.ts

```diff
@@ -1031,6 +1053,12 @@ ${ commands.join('\n') }
         };
       }
     }
+    // Issue 1444 - change name of sudoers file
```

Comment:
A better comment here would be good too (in particular, which versions are relevant).

Reply:
done